content/posts/login-refresh/post.mdx

```diff
@@ -0,0 +1,21 @@
+import post from '@/components/Post';
+import primaryImage from './login.png';
+
+export default post({
+    title: "🔥 Login Redesign",
+    slug: "login-refresh",
+    category: "new",
+    tags: ["design"],
+    authors: ["manu", "ian"],
+    primaryImage,
+    date: new Date("2025-01-27T09:09:58.258Z"),
+});
+
+
+Right now, HCB's login page is a little boring. It gets the job done, but it's by no means exciting. That's why we're thrilled to unveil our refreshed login experience!
+
+It features a sleek, intuitive design and now includes a gallery of handpicked images from successful HCB organizations. Plus, it provides some insight into what HCB is all about.
```

Comment:
```suggestion
It features a sleek, intuitive design and now includes a gallery of handpicked images from iconic organizations running on HCB. Plus, it provides some insight into what HCB is all about.
```

Reply:
Love this! "Successful" implies that other orgs are unsuccessful

Reply:
You got it!